api/v1alpha1/virtualmachine_types.go

```diff
@@ -310,6 +310,9 @@ type VirtualMachineStatus struct {
 	// Guest OS BIOS as a unique hardware identifier.
 	BiosUUID string `json:"biosUUID"`
 
+	// InstanceUUID describes the instance UUID provideed by the underlying vSphere infrastructure.
```

Comment:
`provideed` -> `provided`.

Reply:
Also, should this be `omitempty`?

Reply:
Every VM must have an InstanceUUID (similar to BiosUUID and UniqueID), so will leave the omitempty out.